common/models/meal.json

```diff
@@ -16,11 +16,17 @@
   },
   "validations": [],
   "relations": {
-    "food": {
+    "foods": {
```

Comment:
Based on the other relation json files data, I am skeptical that this is correct